frontend/src/components/EmailSubscriptionContainer.tsx

```diff
@@ -1,7 +1,12 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { subscribeToEmail } from '../services/reviewService';
 
-const EmailSubscriptionContainer: React.FC<{ email: string; setEmail: (email: string) => void; onSubmissionSuccess: () => void; }> = ({ email, setEmail, onSubmissionSuccess }) => {
+interface EmailSubscriptionContainerProps {
+    emailSubmissionHandler: (email: string) => void;
```

Comment:
If you call the backend from `EmailSubscriptionContainer` instead of the `ThankYou` component, you can safely remove the email parameter.
Besides, the `handleEmailSubmissionSuccess` function in the `ThankYou` component does not have any parameter, as it should